cannon/mipsevm/versions/version.go

```diff
@@ -23,8 +23,10 @@ const (
 	VersionMultiThreaded_v2
 	// VersionMultiThreaded64_v3 includes futex handling simplification
 	VersionMultiThreaded64_v3
-	// VersionMultiThreaded64_v4 is the latest 64-bit multithreaded vm, includes support for new syscall eventfd2 and dclo/dclz instructions
+	// VersionMultiThreaded64_v4 includes support for new syscall eventfd2 and dclo/dclz instructions
```

Comment:
nit: I forgot to update this when adding mprotect. While you're here:

```suggestion
	// VersionMultiThreaded64_v4 includes support for new syscalls eventfd2, mprotect and dclo/dclz instructions
```